### PR TITLE
Johnc importvcf docs

### PR DESCRIPTION
**Pull request by johnc1231**
Added docs for importvcf from old markdown docs.

### CODE REVIEW

File: python/hail/context.py [hunk 1]
```diff
@@ -355,6 +355,36 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
                    sites_only=False, store_gq=False, pp_as_pl=False, skip_bad_ad=False, compress=False):
         """Import .vcf files as VariantDataset
 
+        **Notes**
```
Comment (jigold) [inline python/hail/context.py]: Could you please add one basic example?

Something like this:

**Examples**

Import data from a VCF file:

>>> vds = hc.import_vcf('output/example2.vcf.bgz')

File: python/hail/context.py [hunk 2]
```diff
@@ -355,6 +355,36 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+
+        Hail is designed to be maximally compatible with files in the `VCF v4.2 spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
+
+        :py:meth:`~hail.HailContext.import_vcf` takes a list of VCF files to load. All files must have the same header and the same set of samples in the same order
+        (e.g., a dataset split by chromosome). Files can be specified as Hadoop glob patterns.
+
+        Ensuring the VCF files is correctly prepared for import: VCFs should either be uncompressed (".vcf") or block compressed
```
Comment (jigold) [inline python/hail/context.py]: Can you rewrite this sentence -- not grammatically correct.

File: python/hail/context.py [hunk 3]
```diff
@@ -355,6 +355,36 @@ def import_vcf(self, path, force=False, force_bgz=False, header_file=None, npart
...
+        (".vcf.bgz").  If you have a large compressed VCF that ends in ".vcf.gz", it is likely that the file is actually block-compressed,
+        and you should rename the file to ".vcf.bgz" accordingly. If you actually have a standard gzipped file, it is possible to import
+        it to hail using the `force` optional parameter. However, this is not recommended -- all parsing will have to take place on one node, because
```
Comment (jigold) [inline python/hail/context.py]: put force in double backticks.

Comment (jigold) [inline python/hail/context.py]: don't need comma before "because"

**Comment by johnc1231** [comment]
Back to you.

File: python/hail/context.py [hunk 4]
```diff
@@ -346,7 +346,41 @@ def write_partitioning(self, path):
...
+
+        **Notes**
+
+        Hail is designed to be maximally compatible with files in the `VCF v4.2 spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
+
+        :py:meth:`~hail.HailContext.import_vcf` takes a list of VCF files to load. All files must have the same header and the same set of samples in the same order
+        (e.g., a dataset split by chromosome). Files can be specified as Hadoop glob patterns.
+
+        Ensure that the VCF file is correctly prepared for import: VCFs should either be uncompressed (".vcf") or block compressed
+        (".vcf.bgz").  If you have a large compressed VCF that ends in ".vcf.gz", it is likely that the file is actually block-compressed,
```
Comment (jigold) [inline python/hail/context.py]: put these in italics instead of double quotes. (single asterisks)

File: python/hail/context.py [hunk 5]
```diff
@@ -346,7 +346,41 @@ def write_partitioning(self, path):
...
+        and you should rename the file to ".vcf.bgz" accordingly. If you actually have a standard gzipped file, it is possible to import
+        it to hail using the ``force`` optional parameter. However, this is not recommended -- all parsing will have to take place on one node because
```
Comment (jigold) [inline python/hail/context.py]: Capitalize Hail

File: python/hail/context.py [hunk 6]
```diff
@@ -346,7 +346,41 @@ def write_partitioning(self, path):
...
+        **Examples**
+
+        >>> vds = hc.import_vcf('data/example2.vcf.bgz')
+
+        **Notes**
+
+        Hail is designed to be maximally compatible with files in the `VCF v4.2 spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
+
+        :py:meth:`~hail.HailContext.import_vcf` takes a list of VCF files to load. All files must have the same header and the same set of samples in the same order
+        (e.g., a dataset split by chromosome). Files can be specified as Hadoop glob patterns.
```
Comment (jigold) [inline python/hail/context.py]: Can you add the hadoop glob patterns somewhere? Separate page?

**Comment by johnc1231** [comment]
Addressed comments, and we have a hadoop glob patterns page now.

File: python/hail/docs/other_resources.rst [hunk 7]
```diff
@@ -0,0 +1,32 @@
+.. _sec-other_resources:
+
+===============
+Other Resources
+===============
+
+
```
Comment (jigold) [inline python/hail/docs/other_resources.rst]: Add this or something similar. Can't use underscores in section name.

```.. _sec-hadoopglob:```

File: python/hail/docs/other_resources.rst [hunk 8]
```diff
@@ -0,0 +1,32 @@
+.. _sec-other_resources:
+
```
Comment (jigold) [inline python/hail/docs/other_resources.rst]: Either add the SQL section to this page or move the hadoop glob patterns to a separate page. Then this page would look like:

```
.. toctree::
    :maxdepth: 2
 
    Hadoop Glob Patterns <hadoop_glob_patterns>
    SQL <sql>
```

File: python/hail/context.py [hunk 9]
```diff
@@ -346,7 +346,41 @@ def write_partitioning(self, path):
...
+        **Examples**
+
+        >>> vds = hc.import_vcf('data/example2.vcf.bgz')
+
+        **Notes**
+
+        Hail is designed to be maximally compatible with files in the `VCF v4.2 spec <https://samtools.github.io/hts-specs/VCFv4.2.pdf>`_.
+
+        :py:meth:`~hail.HailContext.import_vcf` takes a list of VCF files to load. All files must have the same header and the same set of samples in the same order
+        (e.g., a dataset split by chromosome). Files can be specified as `Hadoop glob patterns <https://hail.is/hail/other_resources.html#hadoop-glob-patterns>`_.
```
Comment (jigold) [inline python/hail/context.py]: Link didn't work for me. It is better to let Sphinx resolve the link:

```
`Hadoop glob patterns <hadoopglob>`_
```

File: python/hail/context.py [hunk 10]
```diff
@@ -346,7 +346,41 @@ def write_partitioning(self, path):
...
+
+        Ensure that the VCF file is correctly prepared for import: VCFs should either be uncompressed (*.vcf*) or block compressed
+        (*.vcf.bgz*).  If you have a large compressed VCF that ends in *.vcf.gz*, it is likely that the file is actually block-compressed,
+        and you should rename the file to ".vcf.bgz" accordingly. If you actually have a standard gzipped file, it is possible to import
+        it to Hail using the ``force`` optional parameter. However, this is not recommended -- all parsing will have to take place on one node because
+        gzip decompression is not parallelizable. In this case, import could take significantly longer.
+
+        Hail makes certain assumptions about the genotype fields, see `Representation <https://hail.is/hail/representation/hail.representation.Genotype.html>`_. On import, Hail filters
```
Comment (jigold) [inline python/hail/context.py]: This links works, but we should try not to have the website path hardcoded. Double check this syntax works locally first:

```
:class:`Representation <hail.representation.Genotype>`
```

**Comment by johnc1231** [comment]
Back to you, probably last time.

File: python/hail/docs/other_resources.rst [hunk 11]
```diff
@@ -0,0 +1,12 @@
+.. _sec-other-resources:
+
+===============
+Other Resources
+===============
+
+.. toctree::
+    :maxdepth: 2
+ 
+    Hadoop Glob Patterns <hadoop_glob_patterns>
```
Comment (jigold) [inline python/hail/docs/other_resources.rst]: I think you forgot to add this file to the repository.

**Comment by johnc1231** [comment]
Thanks for the catch. Fixed

**Pull request merged**